INFO:root:Namespace(FOV=0.2, G_max=40, SNR=False, S_max=200, TSP=True, TSP_epoch=20, a_max=0.17030400000000004, acc_weight=0.1, batch_size=24, checkpoint='summary/32/radial_TSP_0.001_0.1_0.1/model.pt', data_parallel=True, data_path=PosixPath('/mnt/shared/home/b31xs3'), decimation_rate=10, device='cuda', drop_prob=0.0, dt=1e-05, exp_dir='summary/32/radial_TSP_0.001_0.1_0.1', gamma=42576, initialization='radial', interp_gap=1, lr=0.001, lr_gamma=0.01, lr_step_size=30, n_shots=32, num_chans=32, num_epochs=300, num_pools=4, rec_weight=1, report_interval=100, resolution=320, resume=False, sample_rate=1.0, seed=42, sub_lr=0.001, test_name='32/radial_TSP_0.001_0.1_0.1', trajectory_learning='1', v_max=3.40608, vel_weight=0.1, weight_decay=0.0)
Traceback (most recent call last):
  File "train.py", line 530, in <module>
    train()
  File "train.py", line 444, in train
    train_loss, train_time = train_epoch(args, epoch, model, train_loader, optimizer, writer)
  File "train.py", line 205, in train_epoch
    output = model(input)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/parallel/data_parallel.py", line 150, in forward
    return self.module(*inputs[0], **kwargs[0])
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/shared/home/b31xs3/PILOT/models/subsampling_model.py", line 116, in forward
    output = self.reconstruction_model(input)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/shared/home/b31xs3/PILOT/models/rec_models/unet_model.py", line 125, in forward
    output = layer(output)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/shared/home/b31xs3/PILOT/models/rec_models/unet_model.py", line 51, in forward
    return self.layers(input)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/container.py", line 100, in forward
    input = module(input)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/modules/activation.py", line 94, in forward
    return F.relu(input, inplace=self.inplace)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/nn/functional.py", line 914, in relu
    result = torch.relu(input)
RuntimeError: CUDA out of memory. Tried to allocate 300.00 MiB (GPU 0; 15.89 GiB total capacity; 14.99 GiB already allocated; 14.12 MiB free; 15.18 GiB reserved in total by PyTorch)
Traceback (most recent call last):
  File "reconstructe_nosave.py", line 128, in <module>
    reconstructe()
  File "reconstructe_nosave.py", line 108, in reconstructe
    model = load_model(args.checkpoint)
  File "reconstructe_nosave.py", line 60, in load_model
    checkpoint = torch.load(checkpoint_file)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/serialization.py", line 525, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/serialization.py", line 212, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/mnt/shared/home/b31xs3/PILOT/myenv/lib64/python3.6/site-packages/torch/serialization.py", line 193, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'summary/32/radial_TSP_0.001_0.1_0.1/best_model.pt'
